src/main/java/org/jabref/preferences/JabRefPreferences.java

```diff
@@ -760,6 +761,24 @@ private static String convertListToString(List<String> value) {
         return value.stream().map(val -> StringUtil.quote(val, STRINGLIST_DELIMITER.toString(), '\\')).collect(Collectors.joining(STRINGLIST_DELIMITER.toString()));
     }
 
+    private static List<String> convertStringToList(String toConvert) {
+        if (StringUtil.isBlank(toConvert)) {
+            return Collections.emptyList();
+        }
+
+        StringReader reader = new StringReader(toConvert);
+        List<String> result = new ArrayList<>();
+        Optional<String> rs;
+        try {
+            while ((rs = getNextUnit(reader)).isPresent()) {
+                result.add(rs.get());
+            }
+        } catch (IOException ignored) {
+            // Ignored
+        }
```

Comment:
I wouldn't ignore this, because if it fails then preferences may have been corrupted.

Reply:
I just copied it from getStringList. Also Exception occurs only, if the stringreader is fails to read a line in the prefs. Worst case is that a preference option is not loaded correctly and has to be reconfigured manually.